### DISCUSSION TITLE
Why mask as tio.LabelMap not transformed same as image.

### DISCUSSION THREAD

**Discussion by sunshichen** [TorchIO-project/torchio#477] [Q&A] [answered]
I'm trying to use torchio to do augmentation as below:
```python
subjects = []
for image_path, label_path in zip(T1_img_flist, T1_msk_flist):
    subject = tio.Subject(mri=tio.ScalarImage(image_path), brain=tio.LabelMap(label_path))
    subjects.append(subject)
training_transform = tio.Compose([
    tio.ToCanonical(),
    tio.RandomBiasField(p=0.3),
    tio.ZNormalization(masking_method=tio.ZNormalization.mean),
    tio.RandomNoise(p=0.5),
    tio.RandomFlip(),
    tio.RandomAffine(p=1),
    tio.RandomElasticDeformation(p=1,num_control_points=7, max_displacement=15,  locked_borders=2),
])
dataset = tio.SubjectsDataset(subjects, transform=training_transform)
```
However, the brain mask not transform  as MRI image. Is there any problem with my code? How can I acheive that.

**Comment by fepegar** [comment, marked as answer]
Hi, @sunshichen. Your code looks fine. Can you please share two anonymized
images so I can reproduce this?

**Reply by sunshichen** [reply]
Actually, I just found the problem. 🤣  Just a simple silly mistake. 
I called it twice to get image and mask result which caused different transformed results. 
BTW, Thanks greatly for your fast reply! ❤️
